Cryptocurrency/Ethereum/eth_basefee.30s.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+#
+# <xbar.title>Ethereum Base Fee</xbar.title>
+# <xbar.version>v1.0.0</xbar.version>
+# <xbar.author>David Dornseifer</xbar.author>
+# <xbar.author.github>dpdornseifer</xbar.author.github>
+# <xbar.desc>Ethereum EIP1559 base fee.</xbar.desc>
+# <xbar.abouturl>https://ethgasstation.info/</xbar.abouturl>
+# <xbar.dependencies>jq</xbar.dependencies>
```

Comment:
Thanks for this plugin. Please add an image.

Reply:
image added

Reply:
@matryer please review